airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/TemporalClient.java

```diff
@@ -400,7 +400,9 @@ public TemporalResponse<StandardSyncOutput> submitSync(final long jobId, final i
         .withState(config.getState())
         .withResourceRequirements(config.getResourceRequirements())
         .withSourceResourceRequirements(config.getSourceResourceRequirements())
-        .withDestinationResourceRequirements(config.getDestinationResourceRequirements());
+        .withDestinationResourceRequirements(config.getDestinationResourceRequirements())
+        .withConnectionId(connectionId)
+        .withWorkspaceId(config.getWorkspaceId());
```

Comment:
I noticed workspace_id wasn't passed in here. But is passed in other places where we build this struct. @mfsiega-airbyte was that intentional?

Reply:
Indeed not intentional - but same question as Pedro.